src/flexible-text/flexible-text.component.ts

```diff
@@ -72,26 +74,28 @@ export class ThyFlexibleTextComponent implements OnInit, AfterContentInit, OnDes
     }
 
     ngAfterContentInit() {
-        // Wait for the next time period to avoid blocking the js thread
```

Comment:
It is recommended to keep the original notes。

Reply:
Hey, done.